gcs/src/gcs_core.cpp

```diff
@@ -1310,6 +1347,8 @@ ssize_t gcs_core_recv (gcs_core_t*          conn,
             conn->backend.close(&conn->backend);
             if (GCS_ACT_INCONSISTENCY != recv_act->act.type) {
                 /* inconsistency event must be passed up */
+                gu_fatal("Unrecoverable error happened above. Aborting...");
+                usleep(1000000); // give it a second
```

Comment:
why waiting for a second before aborting? Is something going to clean up? If so something less time based would be better.

From: 3ae0ddaed7d23ea36e83a7ab83f85c65493c3ed7

Reply:
I'm not sure but this node will go non-Primary i.e. out of cluster and I think gu_abort() below will produce core dump.